docs/modules/ROOT/pages/explanations/decisions/redis_galera.adoc

```diff
@@ -0,0 +1,64 @@
+= Redis and MariaDB/Galera Provisioning
+
+This page combines the decisions for deploying Redis, MariaDB and Galera, as they are already in use for SPKS.
+
+== Problem
+
+We want to re-use proven methods to deploy Redis and MariaDB/Galera.
+
+== Operators
```

Comment:
Shouldn't we talk about different possibilities here and Operators being one of it?